unstructured/ingest/v2/processes/connectors/mongodb.py

```diff
@@ -32,12 +32,12 @@
 
 @dataclass
 class MongoDBAccessConfig(AccessConfig):
-    uri: Optional[str] = None
+    uri: Optional[str] = enhanced_field(sensitive=True)
 
 
 @dataclass
 class MongoDBConnectionConfig(ConnectionConfig):
-    access_config: MongoDBAccessConfig = enhanced_field(sensitive=True)
+    access_config: MongoDBAccessConfig
```

Comment:
Seems like this _is_ supposed to be enhanced_field(sensitive=True) according to the slack discussion with Roman

Reply:
Alright, put it back!